src/reducers/index.js

```diff
@@ -7,6 +7,30 @@ const dbReducer = (state = {}, action) => {
         return _.extend({ [action.payload.id] : action.payload}, state)
       }
       break;
+    case 'OPTIMISTIC_UPDATE_LOCATION':
+      if(action.payload){
+        return _.extend(
```

Comment:
Here too, looks like we could avoid underscore with something like:
```
return {
  ...state,
  [`last/${action.payload.id}`] : state[action.payload.id],
  [action.payload.id] : {
    ...state[action.payload.id],
    ...action.payload.params,
  },
};
```

Reply:
All set